packages/strapi-plugin-content-manager/admin/src/components/Edit/index.js

```diff
@@ -55,6 +55,40 @@ const getInputType = (type = '') => {
 };
 
 class Edit extends React.PureComponent {
+  state = { shouldUpdateEnumFields: true };
+
+  componentDidUpdate() {
+    if (this.state.shouldUpdateEnumFields && this.props.isCreating) {
+      this.orderAttributes()
+        .forEach(attr => {
+          const details = get(this.props.schema, ['editDisplay', 'availableFields', attr]);
+
+          const layout = this.getInputLayout(attr);
+          const appearance = get(layout, 'appearance');
+          const type = !isEmpty(appearance)
+            ? appearance.toLowerCase()
+            : get(layout, 'type', getInputType(details.type));
+
+          if (type === 'select') {
+            const attribute = get(this.props.schema, ['attributes', attr]);
```

Comment:
Here's my work so far:

```
 const value = attribute.default ? attribute.default : attribute[0];
            // Apparently you can't set the value to either null or '' if this options isn't set in your model

            const target = {
             type: 'select',
             name: attr,
             value,
           };
```

Reply:
Mmm but that's the reason why we try to programmatically add an empty option if the field is not required, right?

From:
<img width="549" alt="screenshot 2018-12-24 at 10 03 47" src="https://user-images.githubusercontent.com/1007051/50395290-3abafb00-0763-11e9-9e93-0e1e87dca1c3.png">

To:
<img width="543" alt="screenshot 2018-12-24 at 10 05 13" src="https://user-images.githubusercontent.com/1007051/50395309-6b9b3000-0763-11e9-9f76-bd69cc6b1aa4.png">

Reply:
Well with the new code we programmatically set the value if the user didn't set a default one in his model.

Reply:
Updated wit your code!